web/src/components/netflow-overview/panel-kebab.tsx

```diff
@@ -58,6 +60,23 @@ export const PanelKebab: React.FC<PanelKebabProps> = ({ id, options, setOptions
     [setOptions, options]
   );
 
+  const onOverviewExport = React.useCallback(() => {
+    const overview_flex = document.getElementById(id);
+
+    setShowOptions(false);
+    if (overview_flex)
+      toPng(overview_flex, { cacheBust: true, backgroundColor: isDark ? '#0f1214' : '#f0f0f0' })
+        .then(dataUrl => {
+          const link = document.createElement('a');
+          link.download = 'overview.png';
```

Comment:
```suggestion
          link.download = `overview_${id}.png`;
```

What do you think about reusing the panel id in the exported file name ?